app/src/main/AndroidManifest.xml

```diff
@@ -96,6 +97,10 @@
 			android:name=".about.ContributorsActivity"
 			android:label="@string/drawer_contributors"/>
 
+		<service
+			android:name="com.mapzen.android.lost.internal.GeofencingIntentService"
+			android:exported="true" />
```

Comment:
Could you explain why this is needed now?

Reply:
Good question. Basically, for Android 12+, any Android components with intent filters require to explicitly declare the `exported`. More info is [here](https://developer.android.com/about/versions/12/behavior-changes-12#exported).
My understanding is that we use this service internally and I found a solution [here](https://github.com/lostzen/lost/issues/272)

Reply:
Does it work without exporting the service? i.e. setting exported to false